docs: Include fix for git+ssh urls with colons (#7582)
meltano/meltano: docs/src/_guide/plugin-management.md

      pip_url: git+ssh://git@gitlab.com/meltano/tap-gitlab.git
      ```

>       Depending on your git provider (such as Azure Repos), some `git+ssh` URLs may contain a colon. This colon can cause errors with `pip`. In this case, try replacing the colon with a forwardslash.

Reviewer: ```suggestion
      Depending on your git provider (such as Azure Repos), some `git+ssh` URLs may contain a colon. This colon can cause errors with `pip`. In this case this can be fixed by replacing the colon with a forward slash.
```

>       i.e. `git+ssh://git@ssh.dev.azure.com:v3/my_org/` becomes `git+ssh://git@ssh.dev.azure.com/v3/my_org/`

Reviewer: Let's vertically align the example before/after URL so that the change is more obvious.

````suggestion

For example, instead of this:
```
git+ssh://git@ssh.dev.azure.com:v3/my_org/
```
Use this:
```
git+ssh://git@ssh.dev.azure.com/v3/my_org/
```
````
